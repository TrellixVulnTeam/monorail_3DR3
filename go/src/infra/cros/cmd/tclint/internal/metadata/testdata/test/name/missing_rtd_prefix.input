{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/tauto",
            "tests": [
                {
                    "name": "remoteTestDrivers/tast/tests/foo"
                }
            ]
        }
    ]
}