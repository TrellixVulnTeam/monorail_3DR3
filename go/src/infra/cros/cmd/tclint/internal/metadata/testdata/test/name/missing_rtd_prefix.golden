[
"## Errors ##",
"Test 'remoteTestDrivers/tast/tests/foo': name must be prefixed with RemoteTestDriver name 'remoteTestDrivers/tauto'"
]